Update Find on page Privacy Whitepaper content (#2839)
MicrosoftDocs/edge-developer: microsoft-edge/privacy-whitepaper/index.md

<!-- ====================================================================== -->
## Find on page 

> Find on page allows you to search a web page for important keywords. Open **Settings and more (...)** > **Find on page** or invoke **Ctrl + F shortcut**. A search box will appear. Type any keyword or phrase into this search box and the page will highlight all locations on the page where your exact search term appears. You can then move through these terms with the **Previous result** and **Next result** buttons.

Reviewer: ```suggestion
Find on page allows you to search a web page for important keywords.  In a webpage, press **Ctrl+F** (Windows, Linux) or **Command+F** (macOS).  Or, click **Settings and more** (**..**), and then select **Find on page**. A search box opens. Type a word or phrase into this search box, and the page highlights all locations on the page where your search string appears. You can then move through these terms by clicking the **Previous result** and **Next result** buttons.
```
* In this case, Ctrl+F is so common, putting it before the GUI approach.
* etc

> Find on page allows you to search a web page for important keywords.  In a webpage, press **Ctrl+F** (Windows, Linux) or **Command+F** (macOS).  Or, click **Settings** (**..**), and then select **Find on page**. A search box opens. Type a word or phrase into this search box, and the page highlights all locations on the page where your search string appears. You can then move through these terms by clicking the **Previous result** and **Next result** buttons.

Reviewer: ```suggestion
Use the "Find on page" feature to search a webpage for key words.  In a webpage, press **Ctrl+F** (Windows, Linux) or **Command+F** (macOS).  Or, click **Settings** (**..**), and then select **Find on page**.  A search box opens.  Enter a word, phrase, or other string into this search box.  As you type, all occurrences of the string in the page are highlighted. You can then move through the search results by clicking the **Previous result** and **Next result** buttons.
```
* "webpage": 1 word per corp style guide.  https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/w/web-world-wide-web-www
* "Use the ..." - per corp style guide, instead of "allows". https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/allow-allows (+ other pages).
* "click" - per internal Edge Dev Style Guide & this repo's conventions.

> By default, the **Find on page** feature sends data to Microsoft to help find other related matches, including synonyms and alternate spellings.

Reviewer 2: ```suggestion
By default, the **Include related matches in Find on page** feature allows **Find on page** to find other related matches, including synonyms and alternate spellings. 

When the **Include related matches in Find on page** toggle is turned on, Microsoft Edge sends the text of the webpage, your search terms, and a service token to a Microsoft cloud service over a secure HTTPS connection. The service token doesn't contain any user-identifiable information. A Microsoft cloud service then processes the text to find results on the page that are related to your search. The webpage text and search terms that are sent to Microsoft are deleted immediately after processing occurs.
```
"When" instead of "if" because it's on by default. Didn't include "no data is stored for any period of time" to remove absolute language.

> Related matches in **Find on page** aren't available in **InPrivate browsing**.

Reviewer: ```suggestion
Related matches in **Find on page** aren't available when using InPrivate browsing.
```

To stop Microsoft Edge from finding related matches in **Find on page**:

> 1. Go to edge://settings/privacy

Reviewer: ```suggestion
1. Go to `edge://settings/privacy`.
```
global comment in this article, may be out of scope but should be noted: docs should give the GUI usage way first, instead of this obscure power-user shortcut URL

you can perhaps Resolve this comment and keep as-is

> 1. Go to edge://settings/privacy

Reviewer: ```suggestion
1. Go to `edge://settings/privacy`.
```
backticks on URLs or quasi-urls, like elsewhere in article

2. In the **Services** section, turn off the **Include related matches in Find on page** toggle.

> If the **Include related matches in Find on page** toggle is turned on, Microsoft Edge sends the text of the webpage, your search terms, and a service token to a Microsoft cloud service over a secure HTTPS connection. The service token doesn't contain any user-identifiable information. A Microsoft cloud service then processes the text to find results on the page that are related to your search. The webpage text and search terms sent to Microsoft are both deleted immediately after processing occurs. No data is stored for any period of time.

Reviewer: ```suggestion
If the **Include related matches in Find on page** toggle is turned on, Microsoft Edge sends the text of the webpage, your search terms, and a service token to a Microsoft cloud service over a secure HTTPS connection. The service token doesn't contain any user-identifiable information. A Microsoft cloud service then processes the text to find results on the page that are related to your search. The webpage text and search terms that are sent to Microsoft are deleted immediately after processing occurs. No data is stored for any period of time.
```
* "that" - in line with https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/that-vs-which - " Include _that_ even if the sentence is clear without it. It helps to clarify the sentence structure."
* "both" - superfluous, removed.

> If the **Include related matches in Find on page** toggle is turned on, Microsoft Edge sends the text of the webpage, your search terms, and a service token to a Microsoft cloud service over a secure HTTPS connection. The service token doesn't contain any user-identifiable information. A Microsoft cloud service then processes the text to find results on the page that are related to your search. The webpage text and search terms that are sent to Microsoft are deleted immediately after processing occurs. No data is stored for any period of time.

Reviewer 2: ```suggestion

```

Reviewer: done locally
